lib/Loop/EvDriver.php

```diff
@@ -125,6 +125,8 @@ public function __destruct() {
         foreach ($this->events as $event) {
             $event->stop();
         }
+
+        $this->events = [];
```

Comment:
Let's add a comment here as to why this is here, because it otherwise seems pointless.